package.json

```diff
@@ -0,0 +1,56 @@
+{
+  "name": "blockingproxy",
+  "description": "WebDriver Proxy for testing rich clients. It block certain calls until Angular is done updating the page under test.",
+  "homepage": "https://github.com/angular/",
+  "keywords": [
+    "test",
+    "testing",
+    "webdriver",
+    "webdriverjs",
+    "selenium"
+  ],
+  "author": "Michael Giambalvo <heathkit@google.com>",
+  "dependencies": {
+    "es6-promise": "^4.0.5",
```

Comment:
I'm trying to use ES6 promises instead of q promises. Unfortunately, we need to target ES5 to support Node v4, even though it already has ES6 promises, thus the polyfill. There might be a better way to do this, though this can also go away when we move to Node v6.